requirements.txt

```diff
@@ -23,3 +23,9 @@ passlib>=1.6.2,<1.7
 lockfile>=0.10.2,<0.11
 python-gnupg>=0.3.7,<0.4
 jsonpath-rw>=1.3.0
+# Requirements for linux pack
```

Comment:
Just a FYI - comments are fine here, but right now we can't use them in st2client requirements.txt since we directly use requirements.txt in the st2client setup.py without ignoring the comments.